community/modules/scheduler/htcondor-configure/README.md

```diff
@@ -72,9 +72,81 @@ example can be found in the [examples README][htc-example].
   - internal_ip
 ```
 
-A full example can be found in the [examples README][htc-example].
+## High Availability
 
-[htc-example]: ../../../../examples/README.md#htcondor-poolyaml--
+This module supports high availability modes of the HTCondor Central Manager and
+of the Access Points. In these modes, the services can be resiliant against
+zonal failures by distributing the services across two zones. To implement
+HA of the Central Manager service, apply the startup script in the example
```

Comment:
Here I am interpreting "apply the startup script in the example" as "have two VMs `use` the startup script generated by `htcondor_central_manager_startup` in the example above. 

My critique here being:
- The use of `apply` feels overloaded and maybe tying to the `use` keyword would avoid confusion.
- The reference to above example could be more explicit.

Reply:
I've modified the surround text to note all the changes and massaged the verbs to something less "loaded"